shell/platform/embedder/tests/embedder_test_context_metal.cc

```diff
@@ -100,6 +100,7 @@ FlutterMetalTexture EmbedderTestContextMetal::GetNextDrawable(
   texture.texture_id = texture_info.texture_id;
   texture.texture =
       reinterpret_cast<FlutterMetalTextureHandle>(texture_info.texture);
+  texture.destruction_callback = nullptr;
```

Comment:
Should this get initialized to nullptr in the struct itself?

Reply:
...or can we not do that in a C struct

Reply:
no - the embedder API declarations are C compatible and do not offer a way to do this automatically.

Reply:
`FlutterMetalTexture texture = {};` should zero out a C struct. In this case we declare the struct on line 98.

Reply:
Changed this to use `texture = {}`